examples/simple/run.py

```diff
@@ -7,17 +7,18 @@
     TwoAgeModelInput, HouseholdPopulation, make_initial_condition)
 from model.specs import DEFAULT_SPEC
 from model.common import SEDURRateEquations
+from model.imports import NoImportModel
 from pickle import load, dump
 # pylint: disable=invalid-name
 
-model_input = TwoAgeModelInput(DEFAULT_SPEC)
-
 # List of observed household compositions
 composition_list = array(
     [[0, 1], [0, 2], [1, 1], [1, 2], [2, 1], [2, 2]])
 # Proportion of households which are in each composition
 comp_dist = array([0.2, 0.2, 0.1, 0.1, 0.1,  0.1])
 
+model_input = TwoAgeModelInput(DEFAULT_SPEC, composition_list, comp_dist)
```

Comment:
I am still a little surprised we need `TwoAgeModelInput` class.

Reply:
We won't actually be using the TwoAgeModelInput class anywhere in the paper version of the code, but it needs to be there for the test to work. I will be writing a new test once it's done. I haven't implemented the new specs/inputs in the simple example, but I will do on the next push.